portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Catherine!</h1>");
+    response.getWriter().println("Hello World! - Catherine");
```

Comment:
nit: prefer to have a better description of what this is.

use constant/structure for this 'greeting/quote', and then use the variable name here `response.getWriter().println(greeting);`

since this will be used by the javascript
`fetch('/data').then(response => response.text())....`, better for readers to quickly understand this "Hello World~ - Catherine" i think.